demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts

```diff
@@ -91,24 +96,33 @@ export class DemoMeetingApp {
     this.defaultBrowserBehaviour = new DefaultBrowserBehavior();
     this.meeting = `READINESS_CHECKER-${uuidv4()}`;
     this.name = `READINESS_CHECKER${uuidv4()}`;
-    this.region = (document.getElementById('inputRegion') as HTMLInputElement).value;
-    //start meeting
+    // Initialize logger and device controller to populate device list
     new AsyncScheduler().start(
       async (): Promise<void> => {
-        let chimeMeetingId = '';
-
-        try {
-          chimeMeetingId = await this.authenticate();
-          this.log(`chimeMeetingId: ${chimeMeetingId}`);
-          const button = document.getElementById("authenticate") as HTMLButtonElement;
-          button.disabled = false;
-        } catch (error) {
-          return error;
-        }
+        await this.initializeLoggerAndDeviceController();
       }
     );
   }
 
+  async startMeetingAndInitializeMeetingReadinessChecker(): Promise<void> {
+    //start meeting
+    let chimeMeetingId = '';
+    try {
+      this.region = (document.getElementById('inputRegion') as HTMLInputElement).value;
+      chimeMeetingId = await this.authenticate();
```

Comment:
We should probably want to add a wait step in the integration tests to wait for the test page to load (e.g. https://github.com/aws/amazon-chime-sdk-js/blob/8ca4b2e8612fde2ae0cc2afd15ecb6720a93930f/integration/js/pages/AppPage.js#L272)

Reply:
Done###